api/src/main/resources/Reference_Application_Concepts-23.xml

```diff
@@ -36,15 +36,15 @@
   <concept concept_id="5622" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="" changed_by="1" date_changed="2016-02-20 05:04:58.0" retired_by="1" uuid="5622AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="71617" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2016-07-29 22:19:54.0" retired_by="1" uuid="71617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="72609" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:47:09.0" retired_by="1" uuid="72609AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="72681" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2011-03-04 13:35:25.0" retired_by="1" uuid="72681AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="73667" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:54:34.0" retired_by="1" uuid="73667AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="77413" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2015-12-30 19:50:22.0" retired_by="1" uuid="77413AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="77414" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2011-03-13 16:56:55.0" retired_by="1" uuid="77414AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="77419" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2006-12-17 00:00:00.0" retired_by="1" uuid="77419AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="80106" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:56:42.0" retired_by="1" uuid="80106AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="84462" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2006-12-17 00:00:00.0" retired_by="1" uuid="84462AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="71617" retired="false" form_text="" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2006-12-17 00:00:00.0" version="" changed_by="1" date_changed="2016-07-29 22:19:54.0" retired_by="1" uuid="71617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
Here, this "form text" is something that @rkorytkowski had deleted in a previous commit : https://github.com/openmrs/openmrs-module-referencemetadata/commit/367afa385676a0cadfe5d5f3cbc3ec0a60792d8d . These are being generated automatically yet again by the Data Export tool

Reply:
I understand. Have you looked at the actual changes that you have made to this file?

Reply:
@dkayiwa  After giving it a proper read, I can conclude that the only "real" changes made to this file are the removal of the concept with ID=165095